src/utils/utils.ts

```diff
@@ -1,6 +1,7 @@
 import { Collection } from "discord.js"
 import emojiRegex from "emoji-regex"
-import { emojiToName } from "gemoji"
+// TODO: ESM only libraries? Ew.
+// import { emojiToName } from "gemoji"
```

Comment:
This causes issues when importing on jest, due to the ESM-only nature of it. Need to look into a better fix but its 3 AM and I can't be bothered looking now.

I should probably look into fixing the need for `.js` on imports, as it's dumb.